packages/react-composites/tests/browser/call/HorizontalGallery.test.ts

```diff
@@ -29,7 +29,7 @@ test.describe('HorizontalGallery tests', async () => {
         remoteParticipants: testRemoteParticipants
       })
     );
-    // Click off page to turn away initial aria label
+    // Click outside of screen to turn away initial aria label
```

Comment:
To be honest, this comment no longer adds that much value beyond the function named `clickOutsideOfPage`. Yay for well named helper functions!

We can just delete (all instances of) this comment.

Reply:
Will do all refactor tasks in a separate PR :)